Bookish.DataAccess/BookishService.cs

```diff
@@ -85,7 +87,39 @@ public IEnumerable<BookLoan> GetLoans(string userId)
                 ORDER BY
 	                Loans.DueDate";
 
-            return dbConnection.Query<BookLoan>(query, new { UserID = userId });
+            return dbConnection.Query<BookLoan>(query, new {UserID = userId});
```

Comment:
Your previous whitespace was correct, the should be whitespace after opening curly bracket and before closing curly bracket 🙂